guides/common/modules/proc_changing-the-mirroring-policy-for-a-repository.adoc

```diff
@@ -0,0 +1,13 @@
+[[changing_the_mirroring_policy_for_a_repository]]
+= Changing the Mirroring Policy for a Repository
+
+
+You can set the mirroring policy for a repository.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Select the required product name.
+. On the *Repositories* tab, click the required repository name, locate the *Mirroring Policy* field, and click the edit icon.
+. From the list, select the required mirroring policy and then click `Save`.
```

Comment:
I am unsure about "required" in this procedure. Maybe "select a product" is sufficient?